docs/source/6_api_reference/api.rst

```diff
@@ -31,5 +34,5 @@ unwrap
 mask
```

Comment:
@po09i shouldn't we call this masking (for consistency with module name)

Reply:
Indeed it would be more consistent